sleepecg/io/nsrr.py

```diff
@@ -119,6 +119,42 @@ def _list_nsrr(
     return files
 
 
+def _download_nsrr_file(
```

Comment:
Is the only difference to `_download_file` the missing `checksum_type` parameter?

Reply:
Almost - in case the checksum doesn't match, this function checks whether the downloaded "file" could be the files-overview page on sleepdata.org (which the request is redirected to in case the token is not valid for the requested file), to give a more helpful error message than "checksum mismatch".